sycl/doc/extensions/SubGroupMask/SubGroupMask.asciidoc

```diff
@@ -21,7 +21,7 @@ IMPORTANT: This specification is a draft.
 
 NOTE: Khronos(R) is a registered trademark and SYCL(TM) and SPIR(TM) are trademarks of The Khronos Group Inc.  OpenCL(TM) is a trademark of Apple Inc. used by permission by Khronos.
 
-This document describes an extension which adds a `group_mask` type.  Such a mask can be used to efficiently represent subsets of work-items in a group for which a given Boolean condition holds.  Group mask functionality is currently limited to groups that are instances of the `sub_group` class.
+This document describes an extension which adds a `sub_group_mask` type.  Such a mask can be used to efficiently represent subsets of work-items in a sub-group for which a given Boolean condition holds.  Sub-group mask functionality is currently limited to groups that are instances of the `sub_group` class.
```

Comment:
```suggestion
This document describes an extension which adds a `sub_group_mask` type.  Such a mask can be used to efficiently represent subsets of work-items in a sub-group for which a given Boolean condition holds.
```

I think you can remove this sentence now -- the name makes it clear that it's sub-group specific, and we don't have any plans to make `sub_group_mask` work with other groups.